editor/edit-post/header/ellipsis-menu/index.js

```diff
@@ -28,6 +29,8 @@ const element = (
 				<ModeSwitcher onSelect={ onClose } />
 				<div className="editor-ellipsis-menu__separator" />
 				<FixedToolbarToggle onToggle={ onClose } />
+				<div className="editor-ellipsis-menu__separator" />
+				<Plugins onToggle={ onClose } />
```

Comment:
This looks odd when no plugins exist:

![no plugins](https://user-images.githubusercontent.com/1779930/34499269-2e68e004-efd3-11e7-856a-db7b5fa4d327.png)

In these cases, it probably should not be shown (or shown with messaging that no plugins exist).